<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeReady</class>
 <widget class="QDialog" name="HomeReady">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Home_HomeReady_btCancelTest">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Home_HomeReady_btPen">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Home_HomeReady_btStart">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeReady_lbBK1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeReady_lbBK2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeReady_lbSampleCode">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeReady_lbSampleInfo">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeReady_lbSampleRemark">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeReady_lbTestName">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeReady_lbTestSerial">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_HomeReady_lbTips">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="Home_HomeReady_cbTestType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
